service/src/test/kotlin/no/nav/su/se/bakover/service/revurdering/RegulerGrunnbeløpServiceImplTest.kt

```diff
@@ -166,29 +160,29 @@ internal class RegulerGrunnbeløpServiceImplTest {
                     )
                 } doReturn sak.kopierGjeldendeVedtaksdata(revurdering.periode.fraOgMed, fixedClock).getOrFail().right()
             },
-        ).let { serviceAndMocks ->
-            val actual = serviceAndMocks.revurderingService.beregnOgSimuler(
-                revurderingId = revurdering.id,
-                saksbehandler = saksbehandler,
-            ).getOrFail().revurdering
-
-            inOrder(
-                *serviceAndMocks.all(),
-            ) {
-                verify(serviceAndMocks.revurderingRepo).hent(argThat { it shouldBe revurderingId })
-                verify(serviceAndMocks.utbetalingService).hentUtbetalinger(argThat { it shouldBe sakId })
-                verify(serviceAndMocks.vedtakService).kopierGjeldendeVedtaksdata(
-                    argThat { it shouldBe sakId },
-                    argThat { it shouldBe revurdering.periode.fraOgMed },
-                )
-                verify(serviceAndMocks.revurderingRepo).defaultTransactionContext()
-                verify(serviceAndMocks.revurderingRepo).lagre(argThat { it shouldBe actual }, anyOrNull())
-                verify(serviceAndMocks.grunnlagService).lagreFradragsgrunnlag(
-                    argThat { it shouldBe revurdering.id },
-                    argThat { it shouldBe revurdering.grunnlagsdata.fradragsgrunnlag },
-                )
-                serviceAndMocks.verifyNoMoreInteractions()
-            }
+        )
+
+        val actual = serviceAndMocks.revurderingService.beregnOgSimuler(
+            revurderingId = revurdering.id,
+            saksbehandler = saksbehandler,
+        ).getOrFail().revurdering
+
+        inOrder(
```

Comment:
Du touchet kanskje bare denne tilfeldig, så er jo ikke så farlig å ta det med i denne endringen, men ut fra navnet på testen her så ville jeg tenkt at det var riktig å bare asserte på resultatet, i stedet for alle in-order verifyene.